docs/DeveloperGuide.md

```diff
@@ -1,12 +1,251 @@
 # Developer Guide
 
+- [Acknowledgements](#Acknowledgements)
+- [Design](#Design)
+  1. [Architecture](#Architecture)
+  2. [Ui component](#Ui-component)
+  3. [Logic component](#Logic-component)
+  4. [Model component](#Model-component)
+  5. [Storage component](#Storage-component)
+  6. [Common classes](#Common-classes)
+- [Implementation](#Implementation)
+- [Product scope](#Product scope)
+- [Target user profile]()
+- [Value proposition]()
+- [User Stories]()
+- [Non-Functional Requirements]()
+
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+* The code package command was inspired by a similar package of [@ivanthengwr](https://github.com/ivanthengwr)'s 
+[command package](https://github.com/ivanthengwr/ip/tree/master/src/main/java/duke/command) in his ip. 
+* The style of document in for the developer guide is inspired by the DG of
+[AY2122S1-CS2113T-T10-3](https://github.com/AY2122S1-CS2113T-T10-3/tp/blob/master/docs/DeveloperGuide.md#purpose-of-this-guide)
+
+## Design
+### Architecture
+### Ui component
+### Logic component
+### Model component
+### Storage component
+### Common classes
+
+## Implementation
+
+This section would focus on explaining the application flow and the specifics on how the features are implemented.
+
+List of Commands
+
+The following sequence diagrams to showcase the list of commands, 
+parser() method will not be reflected in order to improve readability.
+
+1. [Add a passenger feature](#Add-a-passenger-feature)
+2. [Delete a passenger feature](#Delete-a-passenger-feature)
+3. [List passengers feature](#List-passengers-feature)
+4. [Add a flight feature](#Add-a-flight-feature)
+5. [Delete a flight feature](#Delete-a-flight-feature)
+6. [List flights feature](#List-flights-feature)
+
+---
+
+### Add a passenger feature
+
+The command Add a passenger adds passenger details of a particular passenger to the passenger list.
+
+**Overview of relevant classes**
+
+**_NOTE:_** Minimal representation of structure of `AddPassengerCommand`
+
+![class diagram](../docs/ug-diagrams/images/passengerAddCmdClassDiagram.jpg)
```

Comment:
Doesn't seem to be reflected properly on the github page - Do check w this site!   "https://ay2223s1-cs2113-t17-1.github.io/tp/DeveloperGuide.html#List-passengers-feature"

Reply:
Do note that the images do not show up in github pages
<img width="802" alt="Screenshot 2022-10-28 110041" src="https://user-images.githubusercontent.com/88148155/198492747-378d3d33-0948-4cce-ac88-aed0ac897563.png">